lib/dungeonExpWeight.js

```diff
@@ -14,18 +14,19 @@ const getDungeonExpWeight = cataXP => {
     }
 
     let n;
+    let extra;
     let tempLevel = 0;
     if (cataXP < 569809640)
```

Comment:
```suggestion
    if (cataXP < dungeonMaxXP)
```

Reply:
why not use the constant?

Reply:
i dont remember why that was set to that oops